docs/mobile-apps/automated-testing/appium/quickstart.md

```diff
@@ -66,7 +66,18 @@ Set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` as environment variables to avo
 
 ## Step 4: Run the Tests
 
-You must have Java and Maven installed to run this sample test. If you are using an IDE, such as IntelliJ or Visual Studio Code, open the `quickstart-appium-java` project you cloned in step 2, then right-click the `src/test/java/tests/RDC/AppiumIosRdcAppTest` and choose `Run`.
+:::caution
+You must have JDK and Maven installed to run this sample test, and you must have set their paths in your `.bash_profile`, for example:
+```
+## Maven Variables
+export M2_HOME=/Applications/apache-maven-3.8.2
+export M2=$M2_HOME/bin
+
+export PATH=$PATH:$M2:$JAVA_HOME/bin:$PATH
+```
+:::
+
+If you are using an IDE, such as IntelliJ or Visual Studio Code, open the `quickstart-appium-java` project you cloned in step 2, then right-click the `src/test/java/tests/RDC/AppiumIosRdcAppTest` and choose `Run`.
```

Comment:
These instructions are specific to IntelliJ - the steps to run the test are a bit different for Visual Studio Code. 
```suggestion
If you are using an IDE, such as IntelliJ, open the `quickstart-appium-java` project you cloned in step 2, then right-click the `src/test/java/tests/RDC/AppiumIosRdcAppTest` and choose `Run`.
```